### PR TITLE
	Fix for Jira Issues :EMM-1167, EMM-1168

### PR DESCRIPTION
**Pull request by Kamidu**

### CODE REVIEW

File: modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js [hunk 1]
```diff
@@ -265,13 +265,12 @@ function runOperation (operationName) {
 //        payload = operationModule.generatePayload("TemperatureController", operationName, list["TemperatureController"]);
 //        serviceEndPoint = operationModule.getTemperatureControllerServiceEndpoint(operationName);
 //    }
-    console.log(payload);
     if(operationName == "NOTIFICATION"){
```
Comment (harshanL) [inline modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js]: Fix formatting issue in if block above.

File: modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js [hunk 2]
```diff
@@ -64,7 +64,7 @@ var log = new Log();
         if (log.isDebugEnabled()) {
             log.debug("Request sent to IdP");
         }
-        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, please click on the "+
+        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, click on the "+
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js]: Fix formatting issues

File: modules/apps/jaggery/emm/src/emm/modules/sso.js [hunk 3]
```diff
@@ -64,7 +64,7 @@ var log = new Log();
         if (log.isDebugEnabled()) {
             log.debug("Request sent to IdP");
         }
-        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, please click on the "+
+        print("<div><p>You are now being redirected to SSO Provider. If the redirection fails, click on the "+
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/modules/sso.js]: Fix formatting issues

File: modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js [hunk 4]
```diff
@@ -315,7 +315,7 @@ $(document).ready(function () {
             $(errorMsg).text("Notifier frequency is a required field. It cannot be empty.");
             $(errorMsgWrapper).removeClass("hidden");
         } else if (notifierType == notifierTypeConstants["LOCAL"] && !isPositiveInteger(notifierFrequency)) {
-            $(errorMsg).text("Provided notifier frequency is invalid. Please check.");
+            $(errorMsg).text("Provided notifier frequency is invalid. ");
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js]: Fix formatting issues

File: modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js [hunk 5]
```diff
@@ -427,7 +427,7 @@ $(document).ready(function () {
             $(errorMsg).text("Monitoring frequency is a required field. It cannot be empty.");
             $(errorMsgWrapper).removeClass("hidden");
         } else if (!isPositiveInteger(notifierFrequency)) {
-            $(errorMsg).text("Provided monitoring frequency is invalid. Please check.");
+            $(errorMsg).text("Provided monitoring frequency is invalid. ");
```
Comment (kasundananjaya) [inline modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js]: Fix formatting issues

File: modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js (hunk 1 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/operation-bar/public/js/operation-bar.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js (hunk 2 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/emm/src/emm/modules/sso.js (hunk 3 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/modules/sso.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js (hunk 4 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js (hunk 5 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js]: Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158

File: modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js [hunk 6]
```diff
@@ -86,7 +86,7 @@ var log = new Log();
         if (log.isDebugEnabled()) {
             sso.log.debug("Logout request recieved from session id ::: " + sso.sessionId);
         }
-        print("<div><p>You are now redirected to Stratos Identity. If theredirection fails, please click the post " +
+        print("<div><p>You are now redirected to Stratos Identity. If theredirection fails, click the post " +
```
Comment (dakshika) [inline modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js]: Typo theredirection ?

File: modules/apps/jaggery/emm/src/emm/modules/sso.js [hunk 7]
```diff
@@ -86,7 +86,7 @@ var log = new Log();
         if (log.isDebugEnabled()) {
             sso.log.debug("Logout request recieved from session id ::: " + sso.sessionId);
         }
-        print("<div><p>You are now redirected to Stratos Identity. If theredirection fails, please click the post " +
+        print("<div><p>You are now redirected to Stratos Identity. If theredirection fails, click the post " +
```
Comment (dakshika) [inline modules/apps/jaggery/emm/src/emm/modules/sso.js]: typo in "theredirection"

File: modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js (hunk 6 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm-web-agent/src/emm-web-agent/modules/sso.js]: Recommended fixes can be found in PR[1]

[1]. #166

File: modules/apps/jaggery/emm/src/emm/modules/sso.js (hunk 7 above)
Comment (Kamidu) [inline modules/apps/jaggery/emm/src/emm/modules/sso.js]: Recommended fixes can be found in PR[1]

[1]. #166

**Pull request merged**